discord-scripts/invite-management.ts

```diff
@@ -0,0 +1,181 @@
+import { Robot } from "hubot"
+import {
+  Client,
+  TextChannel,
+  VoiceChannel,
+  StageChannel,
+  ApplicationCommandOptionType,
+} from "discord.js"
+
+async function createInvite(
+  discordClient: Client,
+  channelId: string,
+  maxAge: number | string = 86400,
+  maxUses: number | string = 0,
+): Promise<string | null> {
+  if (!discordClient) {
+    throw new Error("Discord client is not initialized.")
+  }
```

Comment:
TypeScript guarantees this won't happen, since `discordClient` is of type `Client`, not `Client | undefined`, `Client | null`, or some combo thereof.